database.py

```diff
@@ -0,0 +1,6 @@
+from pymongo import MongoClient
+from pymongo.collection import Collection
+
+client=MongoClient()
+
+coffees_collection : Collection = client.db.coffees
```

Comment:
```suggestion
coffees: Collection = client.db.coffees
```
this name would be fine too